x/avs/keeper/avs_test.go

```diff
@@ -117,40 +118,41 @@ func (suite *AVSTestSuite) TestUpdateAVSInfo_DeRegister() {
 	avsParams.Action = avstypes.RegisterAction
 	err = suite.App.AVSManagerKeeper.UpdateAVSInfo(suite.Ctx, avsParams)
 	suite.NoError(err)
-	info, err := suite.App.AVSManagerKeeper.GetAVSInfo(suite.Ctx, avsAddres)
-	suite.Equal(avsAddres, info.GetInfo().AvsAddress)
-
+	info, err := suite.App.AVSManagerKeeper.GetAVSInfo(suite.Ctx, avsAddress)
+	suite.Equal(avsAddress, info.GetInfo().AvsAddress)
+	suite.CommitAfter(48*time.Hour + time.Nanosecond)
+	suite.CommitAfter(48*time.Hour + time.Nanosecond)
+	suite.CommitAfter(48*time.Hour + time.Nanosecond)
+	suite.CommitAfter(48*time.Hour + time.Nanosecond)
```

Comment:
Do not type the same line many times. Instead, use a constant in the test and loop over it.

Currently, the unbonding period is set to 2 days. However, you are committing 48 hours * 4 = 8 days. Is that intentional? You should instead run the test after committing 2 days. That way the functionality can be verified.